content/post/monitor-redis-with-grafana-dashboard-in-aws/index.md

```diff
@@ -0,0 +1,360 @@
+appbinding.appcatalog.appscode.com/redis-cluster   kubedb.com/redis   7.0.10    77s
+```
+Let’s check if the database is ready to use,
+
+```bash
+$ kubectl get redis -n demo redis-cluster
+NAME            VERSION   STATUS   AGE
+redis-cluster   7.0.10    Ready    2m34s
+```
+> We have successfully deployed redis in AWS.
+
+
+### Create DB Metrics Configurations
+
+First, you have to create a `MetricsConfiguration` object for database. This `MetricsConfiguration` object is used by Panopticon to generate metrics for DB instances.
+Install `kubedb-metrics` charts which will create the `MetricsConfiguration` object for DB:
+
+```bash
+$ helm search repo appscode/kubedb-metrics --version=v2023.06.19
+$ helm install kubedb-metrics appscode/kubedb-metrics -n kubedb --version=v2023.06.19
+```
+
+### Import Grafana Dashboard
+Here, we will port-forward the `prometheus-grafana` service to access Grafana Dashboard from UI.
+
+```bash
+$ kubectl get service -n default
+NAME                                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                      AGE
+alertmanager-operated                     ClusterIP   None             <none>        9093/TCP,9094/TCP,9094/UDP   9m24s
+kubernetes                                ClusterIP   10.100.0.1       <none>        443/TCP                      37m
+prometheus-grafana                        ClusterIP   10.100.149.171   <none>        80/TCP                       9m29s
+prometheus-kube-prometheus-alertmanager   ClusterIP   10.100.82.140    <none>        9093/TCP                     9m29s
+prometheus-kube-prometheus-operator       ClusterIP   10.100.172.75    <none>        443/TCP                      9m29s
+prometheus-kube-prometheus-prometheus     ClusterIP   10.100.59.16     <none>        9090/TCP                     9m29s
+prometheus-kube-state-metrics             ClusterIP   10.100.182.47    <none>        8080/TCP                     9m29s
+prometheus-operated                       ClusterIP   None             <none>        9090/TCP                     9m24s
+prometheus-prometheus-node-exporter       ClusterIP   10.100.188.82    <none>        9100/TCP                     9m29s
+```
+To access Grafana UI Let's port-forward `prometheus-grafana` service to 3063 
+
+```bash
+$ kubectl port-forward -n default service/prometheus-grafana 3063:80
+Forwarding from 127.0.0.1:3063 -> 3000
+Forwarding from [::1]:3063 -> 3000
+Handling connection for 3063
+
+```
+Now, Go to http://localhost:3063/ you will see a login panel of the Grafana UI, use default credential `admin` as the `Username` and `
+` as the `Password`.
+
+![Grafana Login](grafana-login.png)
+
+After logged in successfuly on Grafana UI, import the json files of dashboards given below according to your choice.
+
+Select Import button from left bar of the Grafana UI
+
+![Import Dashboard](import-dashboard.png)
+
+Upload the json file or copy-paste the json codes to the panel json and hit the load button:
```

Comment:
Copy the desired redis dashboard json file from [here](https://github.com/appscode/grafana-dashboards/tree/master/redis) and paste it to the panel json and hit the load button